arraycontext/context.py

```diff
@@ -1,4 +1,71 @@
 """
+Freezing and thawing
```

Comment:
👍 This is great.  One thing that might be cool is a corresponding blurb in *MIRGE-Com* with a link to this documentation, including some examples of how to use it in *MIRGE-Com* and when it is neeeded.

Reply:
Good idea. I added some anchors we can use to link in 91fdaa6.